src/plugins/orangehrmCorePlugin/Service/ConfigService.php

```diff
@@ -60,6 +60,7 @@ class ConfigService
     public const KEY_MIN_SPECIAL_CHARACTERS = 'auth.password_policy.min_special_characters';
     public const KEY_IS_SPACES_ALLOWED = 'auth.password_policy.is_spaces_allowed';
     public const KEY_DEFAULT_PASSWORD_STRENGTH = 'auth.password_policy.default_required_password_strength';
+    public const KEY_ENFORCE_PASSWORD_STRENGTH = 'authentication.enforce_password_strength';
```

Comment:
```suggestion
    public const KEY_ENFORCE_PASSWORD_STRENGTH = 'authentication.enforce_password_strength';
```
can't we go as the same pattern `auth.password_policy.enforce_password_strength` :thinking: